__tests__/app.test.js

```diff
@@ -52,3 +52,14 @@ describe('/api/topics', () => {
 
     });
 });
+
+describe("/api", ()=>{
+    test("GET: 200 - responds with an object detailing all available endpoints", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body })=>{
+            expect(body.endpoints).toEqual(endpoints)
+        })
```

Comment:
Nice simple solution here to testing this. Students often overcomplicate this ticket